src/agent.ts

```diff
@@ -283,12 +283,14 @@ export class Agent {
       ? JSON.parse(functionCall.arguments)
       : undefined;
 
-    const functionToCall = (functions(
-      this._library,
-      this._client
-    ) as any)[name];
+      const functionToCall = (functions(this._library, this._client) as any)[name];
```

Comment:
Please maintain newlines as it helps with readability since there are few different things happening here .